src/hooks/usePriceFetcher.ts

```diff
@@ -20,68 +20,58 @@ function diaKeysToString(diaKeys: DiaKeys) {
 type PricesCache = { [diaKeys: string]: number };
 
 export const usePriceFetcher = () => {
-  const [pricesCache, setPricesCache] = useState<PricesCache>({});
   const { currentTenant } = useSwitchChain();
-  const { api } = useNodeInfoState().state;
   const { getAllAssetsMetadata } = useAssetRegistryMetadata();
+  const allAssetsMetadata = getAllAssetsMetadata();
+  const diaKeys = allAssetsMetadata.map((asset) => asset.metadata.additional.diaKeys);
 
-  useEffect(() => {
-    if (!api) return;
-
-    const fetchPrices = async () => {
-      const allPrices = await api.query.diaOracleModule.coinInfosMap.entries();
+  const getPriceFromBatchingServer = useCallback(async () => {
+    try {
+      const response = await fetch('https://batching-server.pendulumchain.tech/currencies', {
```

Comment:
Let's maybe move the URL into some other file where we define constants.

Reply:
good idea ✅

Reply:
@ebma Implemented ✅

Reply:
Thanks 👍